src/utils/handleVote.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+
+function handleVote(e, setVoted, idToVote, voter, setVotes) {
+    console.log(e.target)
+    console.log(idToVote)
+    if(e.target.innerText === "^") {
+        setVoted(vote => {
+            return vote + 1
+        })
+        setVotes(votes => {
+            return votes + 1
+        })
+        axios.patch(`https://news-api-9k2x.onrender.com/api/${voter === 'article' ? 'articles' : 'comments'}/${idToVote}`, {inc_votes: 1})
+        .then(() => {
+        }).catch(err => {
+            setVoted(vote => {
+                return vote - 1
+            })
+            setVotes(votes => {
+                return votes - 1
+            }) 
+        })
+    } else if(e.target.innerText === "v") {
+        setVoted(vote => {
+            return vote - 1
+        })
+        setVotes(votes => {
+            return votes - 1
+        })
+        axios.patch(`https://news-api-9k2x.onrender.com/api/${voter === 'article' ? 'articles' : 'comments'}/${idToVote}`, {inc_votes: -1})
+        .then(() => {
```

Comment:
if you're using the exact same axios request for both up and down voting (with the exception of which direction you're voting in) then you are in prime territory to extract this to an api file and reuse the same function. You have to think about DRY code and reusability because this will keep your code de-bloated, efficient and easier to navigate